addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -767,3 +769,10 @@ def action_send_email(self):
                 'default_applicant_ids': self.ids,
             }
         }
+
+    def _get_duration_from_tracking(self, trackings):
+        json = super()._get_duration_from_tracking(trackings)
+        for record in self:
+            if getattr(record, 'refuse_reason_id', False) and getattr(record, 'refuse_date', False):
+                json[record.stage_id.id] -= (datetime.now() - record.refuse_date).total_seconds()
+        return json
```

Comment:
Usage of ``get_attr``: Why wouldn't you get those fields ? They are defined just above.
```suggestion
    def _get_duration_from_tracking(self, trackings):
        result = super()._get_duration_from_tracking(trackings)
        now = datetime.now()
        for applicant in self:
            if applicant.refuse_reason_id and applicant.refuse_date:
                json[record.stage_id.id] -= (now - applicant.refuse_date).total_seconds()
        return json
```

+ it doesn't work if the applicant actually got a ``refuse_reason_id`` and a ``refuse_date``, but is put back into an open stage right ?